modules/ROOT/pages/provisioning-kubevirt.adoc

```diff
@@ -0,0 +1,89 @@
+= Provisioning Fedora CoreOS on KubeVirt
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on any KubeVirt-enabled Kubernetes cluster.
+
+== Prerequisites
+
+Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to a Kubernetes environment with https://kubevirt.io/user-guide/operations/installation/[KubeVirt] installed.
+
+== Referencing the KubeVirt Image
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+
+The image for each stream can directly be referenced from the official registry:
+
+----
+quay.io/fedora/fedora-coreos-kubevirt
+----
```

Comment:
maybe let's just go ahead and make this a bulleted list and list out the three options:

- `quay.io/fedora/fedora-coreos-kubevirt:stable`
- `quay.io/fedora/fedora-coreos-kubevirt:testing`
- `quay.io/fedora/fedora-coreos-kubevirt:next`